src/transformers/models/persimmon/modeling_persimmon.py

```diff
@@ -548,7 +548,6 @@ class PersimmonModel(PersimmonPreTrainedModel):
         config: PersimmonConfig
     """
 
-    # Copied from transformers.models.llama.modeling_llama.LlamaModel.__init__ with LLAMA->PERSIMMON,Llama->Persimmon,PersimmonRMSNorm->nn.LayerNorm,norm->final_layernorm,rms_final_layernorm_eps->layer_norm_eps
```

Comment:
IMO removing this is ok as the init is really just 5 lines and I don't think persimmon and Llama are that closely related?! Wdyt @ArthurZucker 

Don't want to add the attn mask converter here as well